src/NuGetGallery/Areas/Admin/Controllers/ReservedNamespaceController.cs

```diff
@@ -49,6 +54,37 @@ public JsonResult SearchPrefix(string query)
             return Json(results, JsonRequestBehavior.AllowGet);
         }
 
+        [HttpGet]
+        public ActionResult FindNamespacesByPrefix(string prefix)
+        {
+            var namespaces = _reservedNamespaceService
+                .FindAllReservedNamespacesForPrefix(prefix, getExactMatches: false)
+                .OrderBy(x => x.Value)
+                .ThenBy(x => x.IsPrefix)
+                .ToList();
```

Comment:
Nit: I know the Gallery largely uses sync for EF but this could be `async`. I don't feel strongly, feel free to skip

```suggestion
                .ToListAsync();
```

Same comment for line 83

Reply:
I wanted to do this but I see no `ToListAsync` in all of NuGetGallery.sln. This would make testing hard to do now so, given it's admin panel, I will not do this. Thanks for the suggestion!